src/main/java/core/basesyntax/DiffColor.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum DiffColor {
+    blue,
```

Comment:
it's a good practice to write enum values with upper case, like **BLUE , not blue**